### PR TITLE
Fix 'View Source Code' button to make it actually open the URL

### PR DESCRIPTION
**Pull request by byeoon**
Previously, the button would do nothing and clicking it would do nothing.  I updated it so now properly opens the correct link and functions as expected. I also improved a couple lines of text in settings to make it sound better.

(I don't know why it also updates the other plugins in the repo, probably a build thing)

### CODE REVIEW

File: GlobalBadges/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "GlobalBadges",
-    "version": "1.2.6",
+    "version": "1.2.6.1",
```
Comment (domi-btnr) [inline GlobalBadges/manifest.json]: Versions like that will not work with my Updater. Could you change it back to `1.2.6` or bump it to `1.2.7`

File: GlobalBadges/src/components/Settings.tsx [hunk 2]
```diff
@@ -129,7 +126,7 @@ export default ({ settings }: SettingsProps) => {
                 <FormSection title="Updates">
                     <FormRow
                         label="Check for Updates on startup"
-                        subLabel={"Checks automatically for updates when starting the Plugin"}
+                        subLabel={"Checks automatically for updates when starting GlobalBadges"}
```
Comment (domi-btnr) [inline GlobalBadges/src/components/Settings.tsx]: Instead of hard coding the name, use `${name}` which is the name from the manifest

**Review by domi-btnr — CHANGES_REQUESTED** [review]

**Comment by domi-btnr** [comment]
> (I don't know why it also updates the other plugins in the repo, probably a build thing)

Yes. When you build without any args, it'll build all plugins in Debug mode. Means they ignore updates. If you want to build for production, you need to use following Command:

Build a single Plugin for production:
```
npm run build -- --push Pluginfolder/
```

Build all Plugins for production:
```
npm run build -- --workflow_dispatch
```

**Comment by byeoon** [comment]
I have no idea what I did, but the codespace(s - yes I tried using two) that I was using to push these commits just stopped working entirely, and corrupted my work and progress. Tomorrow morning I'll update everything with the reviewed changes and fix everything.

File: GlobalBadges/manifest.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
     "rawUrl": "https://raw.githubusercontent.com/domi-btnr/Enmity-Stuff/main/dist/GlobalBadges.js",
     "sourceUrl": "https://github.com/domi-btnr/Enmity-Stuff/tree/main/GlobalBadges",
     "changelog": [
-        "Fixed Badges not appearing on Profiles"
+        "Fixed Badges not appearing on Profiles",
+        "Fixed Settings repository button not opening and updated descriptions."
     ],
-    "hash": "DEBUG"
-}
+    "hash": "f361d9535ed13801a75ce513dfbfa0652a18cd70"
```
Comment (domi-btnr) [inline GlobalBadges/manifest.json]: Please don't build the files by yourself. I have a workflow for this

**Comment by byeoon** [comment]
These changes should be fixed now, and I also built it for production

**Review by domi-btnr — APPROVED** [review]
I'll remove the Build files, since I build the plugins with the Workflow and will also update the other Plugins to fix this. Thank you :3

**Pull request merged**